bedrock/firefox/views.py

```diff
@@ -513,7 +499,7 @@ def get_template_names(self):
 
         if show_devbrowser_firstrun_or_whatsnew(version):
             template = 'firefox/dev-whatsnew.html'
-        elif version.startswith('42.') or version.startswith('43.'):
+        elif show_42_whatsnew(version):
             template = 'firefox/whatsnew_42/variant-a.html'
```

Comment:
While we're at it, should this template (and associated bundles/files) be renamed? It's not really a variant anymore.